azure-pipelines.yml

```diff
@@ -1,6 +1,6 @@
 variables:
-  ITKGitTag: v5.0.0
```

Comment:
I believe we need to support arbitrary Git tags correctly in the Azure configuration. I will break this change out into a separate topic first.